swe_bench/make_datasets/make_dataset.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# @Author  : stellahong (stellahong@fuzhi.ai)
+# @Desc    :
+import os
+from pathlib import Path
+
+from tqdm.auto import tqdm
+
+from data.inference.const import TESTBED
+from metagpt.logs import logger
+from swe_bench.make_datasets.make_instance import prompt_style_2_edits_only
+from swe_bench.utils.parse_diff import filter_changed_line
```

Comment:
parse_diff  does not exist in swe_bench.utils

Reply:
https://github.com/geekan/MetaGPT/pull/1077